.github/workflows/deploy-vercel-production.yml

```
@@ -37,13 +37,16 @@ jobs:
           cache: pnpm
 
       - name: Install Vercel CLI
+        if: ${{ secrets.VERCEL_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)